public/scss/icons.scss

```diff
@@ -133,6 +133,11 @@
   content: "\e92c";
 }
 
+.dcricon-info::before {
+  content: "\e916";
+  color: rgba(0, 0, 0, 0.5);
```

Comment:
Drop the color specification here. Icons should default to text color. Also, this breaks dark mode.